module-1/assignments/example-table.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
+<table>
+    <Header>
+        <heading>Empno</heading>
+        <heading>Empname</heading>
+        <heading>Salary</heading>
+    </Header>
```

Comment:
these could have been used as names for elements, not to make a whole separate row for it